src/app/organisation/index.njk

```diff
@@ -65,16 +80,48 @@
             $("#organisation_table_row tr").filter(function () {
               $(this).toggle($(this).text().toLowerCase().indexOf(value) > -1)
             });
+            
           });
+        
+          $("#searchUsers").on("click", function () {  
```

Comment:
Question: Why does this code exist e.g. it looks liek it dynmic creates a hidden input value, add it the organisation form and submits that form?
- Why is the input for search not part of the `organisation` form or in a separeate form which points to the same endpoint?

I don't mind the code existing, it is just kinda weird.

Reply:
modified the code. removed the hidden dynamic hidden field.